app/controllers/ontologies_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
 
   def show
     @content_object = :ontology
+    params[:id] = params[:ontology_id] if params[:ontology_id]
```

Comment:
`params[:id]` is somehow set to `params[:reference]` instead of the `ontology_id`. The reference id can still be obtained by using `params[:reference]`